react-app/package.json

```diff
@@ -1,53 +1,54 @@
 {
-  "name": "react-app",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "@material-ui/core": "^4.6.0",
-    "@material-ui/icons": "^4.5.1",
-    "auth0-js": "^9.11.3",
-    "axios": "^0.19.0",
-    "event-source-polyfill": "^1.0.11",
-    "jwt-decode": "^2.2.0",
-    "react": "^16.11.0",
-    "react-character-counter": "^1.0.1",
-    "react-dom": "^16.11.0",
-    "react-image-crop": "^8.4.2",
-    "react-loader-spinner": "^3.1.5",
-    "react-redux": "^7.1.3",
-    "react-router-dom": "^5.1.2",
-    "react-s3": "^1.3.1",
-    "react-scripts": "3.2.0",
-    "react-spring": "^8.0.27",
-    "redux": "^4.0.4",
-    "redux-logger": "^3.0.6",
-    "redux-thunk": "^2.3.0",
-    "typeface-montserrat": "0.0.75",
-    "typeface-roboto": "0.0.75"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  },
-  "devDependencies": {
-    "enzyme": "^3.10.0",
-    "enzyme-adapter-react-16": "^1.15.1"
-  }
+	"name": "react-app",
+	"version": "0.1.0",
+	"private": true,
+	"dependencies": {
+		"@material-ui/core": "^4.9.3",
+		"@material-ui/icons": "^4.9.1",
+		"auth0-js": "^9.12.2",
+		"axios": "^0.19.2",
+		"event-source-polyfill": "^1.0.12",
+		"jwt-decode": "^2.2.0",
+		"react": "^16.12.0",
+		"react-character-counter": "^1.0.1",
+		"react-dom": "^16.12.0",
+		"react-image-crop": "^8.5.0",
+		"react-loader-spinner": "^3.1.5",
+		"react-redux": "^7.1.3",
+		"react-router-dom": "^5.1.2",
+		"react-s3": "^1.3.1",
+		"react-scripts": "3.4.0",
+		"react-spring": "^8.0.27",
+		"redux": "^4.0.5",
+		"redux-logger": "^3.0.6",
+		"redux-thunk": "^2.3.0",
+		"typeface-montserrat": "0.0.75",
+		"typeface-roboto": "0.0.75"
+	},
+	"scripts": {
+		"start": "react-scripts start",
+		"build": "react-scripts build",
+		"test": "react-scripts test",
+		"eject": "react-scripts eject",
+		"scss": "sass --watch 'src/SCSS/main.scss' 'src/CSS/index.css'"
```

Comment:
Please remove this script and install `node-sass`

Reply:
done